app/src/components/settings/settings_view/index.tsx

```diff
@@ -212,57 +221,60 @@ const SettingsViewContainer = (props: Props) => {
   }, [])
 
   return (
-    <SettingsWrapper>
-      <TopContent>
-        <Text>Settings</Text>
-      </TopContent>
-
-      <MainContent>
-        <Row>
-          <Column>
-            <Text>RPC Endpoint</Text>
-            <StatusSection>
-              <StatusBadge status={onlineStatus} />
-              <TextLighter>Status: {onlineStatus ? 'OK' : 'Unavailable'}</TextLighter>
-            </StatusSection>
-          </Column>
-          <FiltersControls>
-            <NodeDropdown currentItem={current} dirty dropdownPosition={DropdownPosition.center} items={filterItems} />
-          </FiltersControls>
-        </Row>
-        {current === dropdownItems.length - 1 && (
-          <Input
-            onChange={event => {
-              setUrl(event.target.value)
-              if (current === dropdownItems.length - 1) setCustomUrl(event.target.value)
-            }}
-            placeholder={'Paste your RPC URL'}
-            value={customUrl}
-          ></Input>
-        )}
-      </MainContent>
-
-      <BottomContent>
-        {props.history && (
-          <ButtonRound
-            onClick={() => {
-              props.history.push('/')
-            }}
-          >
-            Back
-          </ButtonRound>
+    <>
+      <ModalCard>
+        <TopCardHeader>
+          <Row>
+            <Column>
+              <RPCTextWrapper>RPC Endpoint</RPCTextWrapper>
+              <StatusSection>
+                <StatusBadge status={onlineStatus} />
+
+                <TextLighter>Status: {onlineStatus ? 'OK' : 'Unavailable'}</TextLighter>
+              </StatusSection>
+            </Column>
+            <FiltersControls>
+              <NodeDropdown
+                currentItem={current}
+                dirty
+                dropdownPosition={DropdownPosition.center}
+                items={filterItems}
+              />
+            </FiltersControls>
+          </Row>
+        </TopCardHeader>
+
+        {isDropDownActive && (
+          <TopCardHeader borderTop={true} style={{ paddingBottom: '10px' }}>
+            <Row>
+              <Column>
+                <RPCTextWrapper>Custom RPC URL</RPCTextWrapper>
+                <Input
+                  onChange={event => {
+                    setUrl(event.target.value)
+                    if (isDropDownActive) setCustomUrl(event.target.value)
+                  }}
+                  placeholder={'Paste your RPC URL'}
+                  value={customUrl}
+                ></Input>
+              </Column>
+            </Row>
+          </TopCardHeader>
         )}
+      </ModalCard>
 
-        <ButtonRow>
-          <ButtonRound
+      <ButtonContainer style={{ minHeight: isDropDownActive ? '186.2px' : '302px' }}>
```

Comment:
Not good to use inline styles like this this is case for passing prop to component. Active dropdown should be handled differently without having to hardcode height...See in our other components how we did it...